code/datums/wires/vending.dm

```diff
@@ -1,64 +1,79 @@
 /datum/wires/vending
 	holder_type = /obj/machinery/vending
 	proper_name = "Vending Unit"
+	/// Keeps track of which language is selected
+	var/language_iterator = 1
```

Comment:
this seems real fragile

Reply:
Added some crash debug statements in https://github.com/tgstation/tgstation/pull/71643/commits/c7cfb1889af4e784c3bc322e1216f4b1e49c0d50